src/SablierV2LockupDynamic.sol

```diff
@@ -62,6 +62,9 @@ contract SablierV2LockupDynamic is
     /// @dev Sablier V2 Lockup Dynamic streams mapped by unsigned integer ids.
     mapping(uint256 id => LockupDynamic.Stream stream) private _streams;
 
+    /// @dev Sablier V2 Lockup Dynamic streamIds mapped by recipient addresses.
+    mapping(address recipient => uint256[] streamIds) private _streamsByUser;
```

Comment:
Let's rename to `_recipientStreamIds`

Reply:
I prefer `_streamsByUser` - it's clearer because it shows that it's the same stream ids, but filtered by the user address.

The `recipient` key should be renamed to `user`.